apps/src/lib/util/javascriptMode.js

```diff
@@ -47,6 +47,8 @@ export function apiValidateType(
   const validatedTypeKey = 'validated_type_' + varName;
   if (typeof opts[validatedTypeKey] === 'undefined') {
     var properType;
+    var customWarning;
+    var warningMessage;
```

Comment:
nit: not sure if we need to declare `warningMessage` up here if it's only used when we actually output the warning (unless we're using it somewhere else that I'm not seeing?)

Reply:
Thanks! Updated scope of variable. Ran failed unit test locally and found the issue - the test used a reserved pin id (1) which should now fail since no longer can be used - I changed test to use valid pin id (0). I will now work on adding a unit test to test a Maker command with a reserved pin id.